gitrevise/todo.py

```diff
@@ -154,17 +154,19 @@ def edit_todos_msgedit(repo: Repository, todos: List[Step]) -> List[Step]:
          f, fixup <commit> = use commit, but fuse changes into previous commit
          s, squash <commit> = like fixup, but also edit the commit message
          c, cut <commit> = interactively split commit into two smaller commits
-         i, index <commit> = leave commit changes unstaged
+         i, index <commit> = stage the change, i.e. add to the index
```

Comment:
We're not really "staging" the change, as `index` is actually a no-op.

I think I'd prefer something like "leave commit changes staged"